config.js

```diff
@@ -92,7 +92,7 @@ function getModelUrl() {
     if (process.env.MODEL_URL) {
         return process.env.MODEL_URL;
     }
-    const tier = extractTierName();
+    const tier = process.env.TIER?.replace(/[^a-zA-Z\d]/g, '')?.trim();
```

Comment:
On production, we've encountered issues with the DevOps team hardcoding 'prod' or 'production.' If these values are set in the env, the same issue could arise. If the value is empty, should we default to **prod** instead of master?

I know this is a very tricky situation. Hope you understand. 
['https://raw.githubusercontent.com/CBIIT/crdc-datahub-models/', tier || '**prod**', '/cache/content.json']

Reply:
For data model URL,  we don't need to make a special logic for production other then others.   The environment variable TIER is set to "prod",  and BE 3.1.0 uses the env setting for tier.   If set default to "prod", it may cause issue if the environment variable TIER was not set for a new tier for example.

Reply:
I will merge the PR. But, we need to make sure what the env variable is set on production.